src/components/login-form/login-form.test.tsx

```diff
@@ -0,0 +1,42 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import {withHistory, withStore} from '../../utils/mocks/mock-component';
+import LoginForm from './login-form';
+
+describe('Component: LoginForm', () => {
+  it('should render correctly', () => {
+    const emailTestId = 'login-email';
+    const passwordTestId = 'login-password';
+    const { withStoreComponent } = withStore(<LoginForm />);
+    const preparedComponent = withHistory(withStoreComponent);
+
+    render(preparedComponent);
+    const emailInput = screen.getByTestId(emailTestId);
+    const passwordInput = screen.getByTestId(passwordTestId);
+
+    expect(emailInput).toBeInTheDocument();
+    expect(passwordInput).toBeInTheDocument();
+  });
+
+  it('should render correctly when user enter email and password', async () => {
+    const emailTestId = 'login-email';
```

Comment:
```suggestion
    const EMAIL_TEST_ID = 'login-email';
```

Критерий по именованию констант